CHANGELOG.md

```diff
@@ -2,16 +2,19 @@
 
 IMPROVEMENTS:
 
-* Documentation improved for `project` resource to include limitations in Note section. [GH-32]
+* Documentation improved for `project` resource to include limitations in Note section. 
+[GH-32](https://github.com/jfrog/terraform-provider-project/pull/32)
 
 ## 1.0.4 (Mar 22, 2022)
 
 BUG FIXES:
 
-* Project Key validation fixed as per the latest version. [GH-25]
+* Project Key validation fixed as per the latest version. 
+[GH-25](https://github.com/jfrog/terraform-provider-project/pull/25)
```

Comment:
We're doing issue and PR right

Reply:
added

Reply:
I thought `GH-25` meant issue 25? I guess.

Reply:
In AWS example, they mark issues with `#`

Reply:
https://www.terraform.io/plugin/sdkv2/best-practices/versioning#entry-format

> Each bullet also references the corresponding pull request number that contained the code changes, in the format of [GH-####]